charts/opentelemetry-operator/crds/crd-opentelemetrycollector.yaml

```diff
@@ -39,153 +39,86 @@ spec:
       jsonPath: .spec.managementState
       name: Management
       type: string
+    deprecated: true
+    deprecationWarning: OpenTelemetryCollector v1alpha1 is deprecated. Migrate to
+      v1beta1.
```

Comment:
This CRD is still the alpha one, I think we'll need to add the beta one to this folder.  That should happen automatically if https://github.com/open-telemetry/opentelemetry-helm-charts/blob/e5e641f868c00e3019f6126e08d4e34f10fd8fd7/Makefile#L56-L77 is updated.